packages/shared-ui/src/flowHandler/blocks/LoginInitBlock.ts

```diff
@@ -89,7 +89,7 @@ export class LoginInitBlock extends Block<BlockDataLoginInit> {
     return this.#isEnvAffectedByUserGestureDetection();
   }
 
-  async continueWithConditionalUI() {
+  async continueWithConditionalUI({ onLoadingStateChange }: { onLoadingStateChange?: (loading: boolean) => void }) {
```

Comment:
Here we are changing the name of the parameter => I would try to keep the name similar to the parameter name used in loginWithPasskeyChallenge (I would only change it if the existing name does not make sense in the context of continueWithConditionalUI).